src/chains/ethereum_1/ethTransaction.ts

```diff
@@ -454,37 +456,49 @@ export class EthereumTransaction implements Transaction {
   /** Get the suggested Eth fee (in GWEI) for this transaction */
   public async getSuggestedFee(
```

Comment:
Confirm that all fees are expressed in native chain currency - e.g. string in Eth '.0000000000000000001', etc.